src/app/shared/utils/date-formats.ts

```diff
@@ -11,6 +11,7 @@ export const DEFAULT_DATE_LOCALE = enGB;
 
 export const DEFAULT_DATE_OUTPUT_FORMAT = 'yyyy-MM-dd';
 export const DEFAULT_DATE_INPUT_FORMAT = 'yyyy-MM-dd';
+export const DATE_INPUT_FORMAT_HINT = 'YYYY-MM-DD';
```

Comment:
Wouldn't this be repeating code unnecessarily? We could just convert the format used to uppercase

Reply:
The idea was to make it more flexible, e.g. the hint could also be "Please enter data as YYYY-MM-DD".

Reply:
Also makes the configured values more obvious/visible.